Automattic/wp-calypso
# Odyssey Stats: Enable email sections and pages

Related to #87399


## Proposed Changes

* Enabled Email Stats card on Traffic page
* Enable Email Stats card on Subscriber page
* Enabled email list page, which lists recent emails
* Enabled email detail page, including Clicks and Opens
* Refactored support checks to a selector
* **Enable Email Stats for all site types on Calypso Stats, including Jetpack sites**

More information about Odyssey Stats testing instructions: PCYsg-Pp7-p2

## Testing Instructions

* Checkout branch `update/enable-email-stats-for-odyssey` and build Jetpack plugin - https://github.com/Automattic/jetpack/pull/35703
* Spin this change up on the local Jetpack environment: `cd apps/odyssey-stats && STATS_PACKAGE_PATH=/path-to-jetpack/projects/packages/stats-admin yarn dev`.
* Open Traffic page
* Ensure Email section works fine
* Click View details
* Ensure the Email list works okay
* Click an individual Email
* Ensure you are taken to Email Opens
* Ensure Email Opens page works okay

**!!Please also test with Jetpack trunk (older Jetpack), ensure the Email sections are still hidden, and it generally looks okay!!**

**!!Please test with Calypso Live Branch, and ensure Calypso Stats still works okay!!**

<img width="1293" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/9e13f9e8-9bc4-4a15-9a37-66d360fb595f">


## Pre-merge Checklist





cc @Automattic/apex @Automattic/explorers @Automattic/gold @Automattic/zap @Automattic/red @lupus2k
